### PR TITLE
enh(UI): Enhance graph styles on Graph widget

### PR DESCRIPTION
**Pull request by Thebarda**
## Description

This adds a dozen of properties about graph styles for the graph widget.

https://github.com/centreon/centreon/assets/12515407/1fe9d4b7-7469-49af-a72b-c57d37c3fb8a

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [x] master

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: centreon/lang/de_DE.UTF-8/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -16858,7 +16858,7 @@ msgstr "Erweiterter Modus"
 # msgid "You are not allowed to list contact templates"
 # msgstr ""
 
-# msgid "Graph style"
+# msgid "Graph styles"
```
Comment (smau-centreon) [inline centreon/lang/de_DE.UTF-8/LC_MESSAGES/messages.po:16861]:
```suggestion
# msgid "Graph style"
```

File: centreon/lang/es_ES.UTF-8/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -17192,7 +17192,7 @@ msgstr "Modo avanzado"
 # msgid "You are not allowed to list contact templates"
 # msgstr ""
 
-# msgid "Graph style"
+# msgid "Graph styles"
```
Comment (smau-centreon) [inline centreon/lang/es_ES.UTF-8/LC_MESSAGES/messages.po:17195]:
```suggestion
# msgid "Graph style"
```

File: centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -18982,8 +18954,8 @@ msgstr "Toutes les vues métier sont sélectionnées"
 msgid "You are not allowed to list contact templates"
 msgstr "Vous n'êtes pas autorisé à lister les modèles de contact"
 
-msgid "Graph style"
-msgstr "Style du graphe"
+msgid "Graph styles"
```
Comment (smau-centreon) [inline centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:18957]:
```suggestion
msgid "Graph style"
```

File: centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -18982,8 +18954,8 @@ msgstr "Toutes les vues métier sont sélectionnées"
...
+msgstr "Styles du graphe"
```
Comment (smau-centreon) [inline centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:18958]:
```suggestion
msgstr "Style du graphe"
```

File: centreon/lang/pt_BR.UTF-8/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -17660,7 +17660,7 @@ msgstr ""
 # msgid "You are not allowed to list contact templates"
 # msgstr ""
 
-# msgid "Graph style"
+# msgid "Graph styles"
```
Comment (smau-centreon) [inline centreon/lang/pt_BR.UTF-8/LC_MESSAGES/messages.po:17663]:
```suggestion
# msgid "Graph style"
```

File: centreon/lang/pt_PT.UTF-8/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -17640,7 +17640,7 @@ msgstr "Você não salvou as alterações, deseja continuar?"
 # msgid "You are not allowed to list contact templates"
 # msgstr ""
 
-# msgid "Graph style"
+# msgid "Graph styles"
```
Comment (smau-centreon) [inline centreon/lang/pt_PT.UTF-8/LC_MESSAGES/messages.po:17643]:
```suggestion
# msgid "Graph style"
```

**Review by smau-centreon — CHANGES_REQUESTED** [review]

**Review by smau-centreon — APPROVED** [review]

File: centreon/packages/ui/src/Graph/LineChart/Legend/LegendHeader.tsx [hunk 7]
```diff
@@ -36,7 +40,9 @@ const LegendHeader = ({
   const legendName = legend || name;
 
   return (
-    <div className={classes.container}>
+    <div
+      className={cx(!isListMode ? classes.container : classes.containerList)}
```
Comment (quborg) [inline centreon/packages/ui/src/Graph/LineChart/Legend/LegendHeader.tsx:44]: pas besoin de cx()

**Review by Noha-ElAbrouki — DISMISSED** [review]

**Review by quborg — DISMISSED** [review]

**Review by smau-centreon — APPROVED** [review]

**Review by Noha-ElAbrouki — APPROVED** [review]

**Comment by Thebarda** [comment]
@coderabbitai resolve all conversations

**Pull request merged**